guides/development-workflow.md

```diff
@@ -0,0 +1,341 @@
+
+### Reviewing another developer's pull request
+
+1. move the Trello card into the 'Under code review' column
+2. find and read the Trello card's acceptance criteria
+3. find and read through the pull request, carrying out a [code review](#code-review)
+  * if it is ready:
+    - press the merge button
+    - delete the leftover branch
+    - move the Trello card into 'Deployed to Staging'
+  * if it's not ready:
+    - leave feedback
+    - mention the author so they're aware
+
+---
+## Working on a ticket
+
+Tickets can fall into two types of work flow depending on their severity.
+
+Most commonly tickets will simply fall into the same [story process](#the-lifecycle-of-a-story), where fixes will go through `develop` and Staging for review and acceptance.
+
+They can however be sometimes when there's an emergency which requires a change to production ASAP. This is when we reach for a [hotfix](#hotfix).
+
+### Hotfix
+
+We think of hotfixes in the same way as [Wikipedia](https://en.wikipedia.org/wiki/Hotfix) describes them:
+
+> a hotfix implies that the change may have been made quickly and outside normal development and testing processes
+
+> A hotfix is a change deemed critical enough that it cannot be held off until a regular content patch.
+
+An emergency for would be when:
+
+1. production is unusable - users are directly affected
+2. a serious vulnerability has been discovered and is currently exploitable
+
+### Issue a hotfix
+When the decision to make a hotfix is made it's important to communicate well. Involve the rest of the team at this point and discuss the plan of action, allowing delivery managers to communicate back to the client.
+
+This is the only time a branch should be based off of `master` instead of `develop`.
+
+  1. create a new card in Trello and move it to 'In progress'
+  2. create a new branch off of `master`
+
+  ```
+  git checkout -b <branch-name> origin/master
+  ```
+
+  3. [commit](#committing) the fix
+  4. make a [pull request](#pull-requests) into `master`
+  5. move the Trello card into 'Awaiting review'
+
+### Review and deploy a hotfix
+  1. review the code
+  2. check the fix is tested locally if possible
+  3. merge the pull request
+  4. manually merge the pull request back into develop
+
+  ```
+  git checkout develop
+  git pull
+  git merge origin/master
+  git push
+  ```
+
+  5. move the card on Trello into 'Deployed to production'
+
+---
+
+## Deploying
+
+Our approach is one of [continuous delivery](https://en.wikipedia.org/wiki/Continuous_delivery) meaning we build, test and deploy as frequently as possible.
+
+We do this by using services such as Chef or Codeship which are configured to listen for Git pushes and then initiate the correct deployment.
+
+### Staging
+Staging deployments are automatic and should be initiated when each a pull request is merged. When you click merge the changes are immediately deployed to the Staging environment.
```

Comment:
"when each a pull request is merged"

There might be an extra extra word in here.